chainer/functions/activation/relu.py

```diff
@@ -12,48 +12,103 @@
     _mode = cudnn.cudnn.CUDNN_ACTIVATION_RELU
 
 
-class ReLU(function.Function):
+class ReLU(function_node.FunctionNode):
 
     """Rectified Linear Unit."""
     # TODO(beam2d): Implement in-place version.
```

Comment:
It is not directly releated to this PR, but I found this TODO comment is obsolete. Can you remove it?